vochain/process.go

```diff
@@ -64,9 +64,15 @@ func (v *State) Process(pid []byte, isQuery bool) (*models.Process, error) {
 	var err error
 	v.RLock()
 	if isQuery {
-		processBytes = v.Store.ImmutableTree(ProcessTree).Get(pid)
+		processBytes, err = v.Store.ImmutableTree(ProcessTree).Get(pid)
+		if err != nil {
+			return nil, err
+		}
 	} else {
-		processBytes = v.Store.Tree(ProcessTree).Get(pid)
+		processBytes, err = v.Store.Tree(ProcessTree).Get(pid)
```

Comment:
Use `if processBytes, err = .... ; err != nil {} `

Reply:
Same for all other cases

Reply:
Right! Updated.

Reply:
Would love to have a linter for this hehe